src/query/sql/src/planner/binder/ddl/account.rs

```diff
@@ -55,7 +55,7 @@ impl Binder {
                 // ALL PRIVILEGES have different available privileges set on different grant objects
                 // Now in this case all is always true.
                 let grant_object = self.convert_to_grant_object(level).await?;
-                let priv_types = grant_object.available_privileges();
+                let priv_types = grant_object.available_privileges_without_ownership();
```

Comment:
how about just `available_privileges` ?

ownership in our design is not considered as a real privilege. we can put a description about this in the comment of this function.